Controller/CartWishlist.controller.js

```diff
@@ -0,0 +1,233 @@
+const usersdb=require('../Models/users.model');
+const cartsdb=require('../Models/carts.model');
+const wishlistsdb=require('../Models/wishlists.model');
+const productsdb=require('../Models/products.model');
+
+module.exports.addToCart=async (req,res)=>{
+    const {id}=req.params;
+    const {productId}=req.body;
+    try{
+        const user=await usersdb.findById(id);
+        const cart=await cartsdb.findById(user.cart);
+        if(cart){
+            if(!cart.cartItems.some(({product})=>product==productId)){
+                cart.cartItems.push({
+                    product:productId,
+                    quantity:1
```

Comment:
you should not hardcode the value of quantity

Reply:
Hey Kunal I'll refactor it!